src/renderer/views/Channel/Channel.vue

```diff
@@ -341,6 +350,15 @@ const isElementListLoading = ref(false)
 const isSearchTabLoading = ref(false)
 const currentTab = ref('videos')
 
+const hideWatchedToggle = ref(false)
```

Comment:
It's not storing an element but the value
```suggestion
const hideWatchedEntries = ref(false)
```

Reply:
Done - commit https://github.com/FreeTubeApp/FreeTube/pull/7366/commits/59e6b1a8205ae0179873b206350f665e6f0292ae